src/lib/autoLauncher.ts

```diff
@@ -150,22 +150,25 @@ export class AutoLauncher {
     if (!this.enabled) {
       console.error("ENABLE DID NOT WORK")
     } else {
-      appConfig.updateAppConfig(null, null, true, null, null)
+      await appConfig.update(null, null, true, null, null)
```

Comment:
not critical, but maybe it will be more readable to pass an object, and instead of `null`s we just provide values for properties we need. In this case, it would be:
```
{ launchOnBoot: true }
```

Reply:
wonderful! implementing now